spec/integration/interactive_mode_spec.rb

```diff
@@ -99,24 +99,46 @@ def expect_to_run_fzf_suggestion(with_pattern:)
     let(:codeowners) { ['lib/new_file.rb @mpospelov @foobar'] }
     let(:owners) { ['@mpospelov'] }
     let(:file_tree) { { 'lib/new_file.rb' => 'bar' } }
-    let(:answers) { ['r', '@mpospelov', 'y'] }
+    let(:answers) { ['r', '@mpospelov', 'a'] }
 
     it 'asks to add new owner to owners' do
       question = <<~QUESTION
         Unknown owner: @foobar for pattern: lib/new_file.rb. Add owner to the OWNERS file?
-        (y) yes
+        (a) add a new owner
         (r) rename owner
         (i) ignore owner in this session
         (q) quit and save
       QUESTION
       expect(runner)
         .to ask(question)
-        .limited_to(%w[y r i q])
+        .limited_to(%w[a r i q])
         .and ask('New owner: ')
         .and ask('Commit changes?')
       expect(codeowners_file_body).to eq("lib/new_file.rb @mpospelov\n")
       expect(owners_file_body.scan('@mpospelov')).to contain_exactly('@mpospelov')
     end
+
+    context 'when the owner`s name was misspelled' do
+      let(:owners) { %w[@mpospelov @foobaz] }
+
+      it 'asks to add new owner to owners' do
+        question = <<~QUESTION
+          Unknown owner: @foobar for pattern: lib/new_file.rb. Did you mean @foobaz? Add owner to the OWNERS file?
+          (c) correct to @foobaz
```

Comment:
while testing locally the first time I read "Did you mean?" than I thought, my answer should be "y" or yes. Maybe we can use yes for this case as the favorite preference.

Reply:
I agree. I just wanted to rename this option to avoid choosing by keeping the meaning from the previos version in mind, cause in the previous version `(y)` stand for adding a new owner to the file.
But the option `y` is  definitely more understandable. So I renamed it.